modules/update-process-workflow.adoc

```diff
@@ -38,8 +36,6 @@ Cluster Operators are updated in separate stages called Runlevels, and the CVO e
 When the CVO deployment is applied, the current CVO pod terminates, and a CVO pod using the new version starts.
```

Comment:
```suggestion
When the CVO deployment is applied, the current CVO pod stops, and a CVO pod that uses the new version starts.
```
Again, I know this line wasn't part of your changes, but a couple of small edits can improve it. Just for your consideration!

See IBM Style Word Usage for guidance about "terminates" and "using": https://www.ibm.com/docs/en/ibm-style?topic=word-usage